### PR TITLE
Add difficulty + overall options to script

### PR DESCRIPTION
**Pull request by kewbish**
This PR passes the `difficulty` and `overall` course quality options to the worker.

(I also ran auto-format on this, the only significant diff is [here](https://github.com/ubccsss/ubccsss.org/compare/master...kewbish:em-add-difficulty-overall-params?expand=1#diff-05b2be1274aac294fead24df7d2d56212cc5f0cc9c8d6926c3706e8424737ea5R51-R58).)

### CODE REVIEW

**Comment by AnimeAllstar** [comment]
test

File: assets/js/create-github-issue.js [hunk 1]
```diff
@@ -71,18 +79,22 @@ const createGithubIssue = async () => {
           user: user,
           review: review,
           reference: reference,
+          difficulty,
```
Comment (AnimeAllstar) [inline assets/js/create-github-issue.js:82]: `difficulty: difficulty`?

**Review by AnimeAllstar — APPROVED** [review]

File: assets/js/create-github-issue.js (hunk 1 above)
Comment (kewbish) [reply assets/js/create-github-issue.js:82]: It's [shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#syntax) so it evaluates to the same thing, but can change for consistency with the other fields!

**Comment by AnimeAllstar** [comment]
Works, just tested it: https://github.com/csssbot/testing/issues/62

**Comment by kewbish** [comment]
Nice, thank you!

**Pull request merged**